N/NEO/build_tarballs.jl

```diff
@@ -7,17 +7,27 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 
 name = "NEO"
-version = v"24.26.30049"#.6
+version = v"25.27.34303"#.5
 
 # Collection of sources required to build this package.
 sources = [
     GitSource("https://github.com/intel/compute-runtime.git",
-              "e16f47e375e4324dae07aadbfe953002a1c45195"),
+              "d0fdeb0339afaa6db37411e10c41f291945aa727"),
+    # patches
+    DirectorySource("./patches"),
```

Comment:
Just a nit, but we typically do `bundled/patches/*.patch` so that `ocloc_wrapper.sh` could go in `bundled` (not being a patch, that is).